app_migrations/64a037bb5344.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+-- Running upgrade 4f0b36e6cf28 -> 64a037bb5344
```

Comment:
Update the old revision number here
```suggestion
-- Running upgrade 286343296975 -> 64a037bb5344
```